app/serializers/rest/application_serializer.rb

```diff
@@ -1,12 +1,15 @@
 # frozen_string_literal: true
 
 class REST::ApplicationSerializer < ActiveModel::Serializer
-  attributes :id, :name, :website, :scopes, :redirect_uri,
-             :client_id, :client_secret
+  attributes :id, :name, :website, :scopes, :redirect_uris,
+             :client_id
```

Comment:
I had to retain `:client_id` here, since it [was previously exposed](https://github.com/mastodon/mastodon/commit/7c3fea727583dd9bf482729cc87272e976a25b2d) on `/api/v1/apps/verify_credentials`, even though it's not strictly necessary for it to be exposed there. (The addition of `client_id` to `ApplicationSerializer` has gone out in a release potentially, even though it appears to only be on `main`

Reply:
Actually, I've removed `client_id` here, since that commit was never in a tagged release from what I can tell.

Reply:
What was the rationale for including `client_id` previously, and why did you decide to stop exposing it?